web/src/input_pill.ts

```diff
@@ -455,6 +466,10 @@ export function create<T>(opts: InputPillCreateOptions<T>): InputPillContainer<T
             store.removePillFunction = callback;
         },
 
+        onTextChange(callback) {
+            store.onKeyDownInputFunction = callback;
+        },
```

Comment:
What's the thinking behind have two different names for this concept? 

I guess `onPillRemove` makes the same mistake, but we should follow the convention of `onPillCreate`, which makes it way easier to read/trace the code. Can you add a bonus commit renaming `removePillFunction` to `onPillRemove` too?

Reply:
Yes. I saw naming convention of onPillRemove and tried naming like that.